### PR TITLE
Update tables

### PR DESCRIPTION
**Pull request by Timowest12**
In this excercise i learned to query multiple databases:

- Created 2 more databases for testing practice
- Modified them to use primary key
- Inserted owners
- Linked owners to animals
- Wrote queries to go trough both databses

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -13,3 +13,41 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,eight_kg) VALUE
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,eight_kg) VALUES('Angemon',date '2005-04-12',1,FALSE,-45);
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,eight_kg) VALUES('Boarmon',date '2005-6-07',7,TRUE,20.4);
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,eight_kg) VALUES('Blossom',date '1998-10-13',3,TRUE,17);
+
+
```
Comment (SajjadAhmad14) [inline data.sql:17]: Please modify the `animals` table to meet the following requirements:
 - If an animal's name ends with `mon` it should have `Digimon` as a `species_id`, all other animals should have `Pokemon` instead.
 - All animals got owners assigned as `Sam Smith owns Agumon`.

File: data.sql [hunk 2]
```diff
@@ -13,3 +13,41 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,eight_kg) VALUE
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,eight_kg) VALUES('Angemon',date '2005-04-12',1,FALSE,-45);
```
Comment (SajjadAhmad14) [inline data.sql:13]: Please add the project title and project features description in README. This way it looks more professional.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Timo, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by rukundoeric** [comment]
Hi @Timowest12, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [x] Your branch has a conflict with the base branch. Kindly fix the conflict before requesting another code review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [x] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: data.sql (hunk 2 above)
Comment (Timowest12) [reply data.sql:13]: added features into readme

**Comment by Timowest12** [comment]
Merge conflict fixed

**Review by descholar-ceo — APPROVED** [review]
Hi @Timowest12 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**